examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3161,6 +3173,8 @@ var GLTFLoader = ( function () {
 
 			assignExtrasToUserData( node, nodeDef );
 
+			parser.associations.set( node, { type: 'nodes', index: nodeIndex } );
```

Comment:
nit: this line seems a bit out of place in the middle of the `nodeDef` assignments, perhaps move it below the matrix/TRS block?

Reply:
Done